grow_cluster_test.py

```diff
@@ -53,6 +53,11 @@ def grow_cluster(self, cluster_target_size):
         nodes_to_add = cluster_target_size - self._cluster_starting_size
         duration = 10 * nodes_to_add
         stress_queue = self.run_stress_thread(duration=duration)
+
+        # Wait for cluster is filled with data
+        # Set space_node_treshold in config file for the size
+        self.db_cluster.wait_total_space_used_per_node()
```

Comment:
The logic is:
   start c-s so we can inject data into the cluster
   wait until N bytes of data is injected
no?